cmd/cost/create.go

```diff
@@ -0,0 +1,72 @@
+package cost
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/costexplorer"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		//OU Flag
+		OUid, err := cmd.Flags().GetString("ou")
+		if err != nil {
+			log.Fatalln("OU flag:", err)
+		}
+
+		//Get Organizational Unit
+		OU := organizations.OrganizationalUnit{Id: aws.String(OUid)}
+		//Initialize AWS clients
+		org, ce := initAWSClients()
+
+		createCostCategory(&OUid, &OU, org, ce)
+	},
+}
+
+func init() {
+	createCmd.Flags().String("ou", "", "get OU ID")
+	err := createCmd.MarkFlagRequired("ou")
+	if err != nil {
+		log.Fatalln("OU flag:", err)
+	}
```

Comment:
It would be great if you make createCmd a function and initialize all the flags at that function.